src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,20 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+
+    /**
+     * Converts voltage to compensated power
+     *
+     * The voltage compensation saturation will determine what voltage represents 100% output.
+     *
+     * The compensated power is the power resulting from turning the voltage off and on without stopping.
+     * It's basically the voltage divided by the saturation of the compensation.
```

Comment:
Not good enough
You need to explain in general the point of voltage compensation.
Then for each parameter explain its meaning.

Reply:
IDK